src/NzbDrone.Core/Tv/ShouldRefreshSeries.cs

```diff
@@ -28,6 +29,19 @@ public bool ShouldRefresh(Series series)
                 return true;
             }
 
+            var episodes = _episodeService.GetEpisodeBySeries(series.Id);
+
+            var atLeastOneAiredEpisodeWithoutTitle = episodes.Any(e =>
+                e.SeasonNumber > 0 &&
+                e.AirDateUtc.HasValue && e.AirDateUtc.Value.Before(DateTime.UtcNow) &&
+                e.Title.Equals("TBA", StringComparison.Ordinal));
+
+            if (atLeastOneAiredEpisodeWithoutTitle && series.LastInfoSync < DateTime.UtcNow.AddHours(-1))
```

Comment:
I think we could just do the refresh regardless if an aired episode has a TBA title, that way at least the automatic update tries to refresh even if they just refreshed manually. Should be a small window where it was manually refreshed < 1 hour ago anyways.

Reply:
Just remove the `series.LastInfoSync < DateTime.UtcNow.AddHours(-1)` and update the log message